docs/_static/scss/_general.scss

```diff
@@ -857,7 +857,7 @@ code.docutils.literal.notranslate {
         margin: 0 20px 0 60px;
         width: 100%;
         max-width: calc(100% - 90px);
-        overflow: scroll;
+        overflow: auto;
```

Comment:
I left this one accidental but was going to be apart of another code change another time, removes this scrollbar that is visible on smaller viewsizes on non-apple devices:

![image](https://github.com/user-attachments/assets/86669b6e-6c40-4931-8967-fc72fc44f1c1)